javascript/src/main/components/UserMessages/UserMessageList.js

```diff
@@ -1,13 +1,67 @@
 import React from "react";
 import BootstrapTable from 'react-bootstrap-table-next';
+import {useAuth0} from "@auth0/auth0-react";
+import useSWR from "swr";
+import {fetchWithToken} from "../../utils/fetch";
 
-export default ({ messages = [] }) => {
+const replaceMessage = (text, slackUsers) => {
+    return text.replace(/<@([A-Z0-9]{11})>/g, (_,userId) => {
+        if(slackUsers != null) {
+            for(let i = 0; i < slackUsers.length; i++) {
+                if(slackUsers[i].id === userId) {
+                    return "@" + slackUsers[i].real_name;
+                }
+            }
+        }
+        return "@" + userId;
+    })
+}
+
+const formatBracketedText = (text) => {
+    var bracketRegEx = /<(.*?)>/g;

```

Comment:
Replace instances of "var" with "let." Helps with scope issues according to LAs.